### DISCUSSION TITLE
How to add new project to docker local hosted uptrace project.

### DISCUSSION THREAD

**Discussion by rupeshkumar1995** [uptrace/uptrace#173] [Q&A] [answered]
Hi I am unable to find how to create a new project inside the locally hosted docker uptrace.
I am new to uptrace.
could some guide for the same.

**Comment by vmihailenco** [comment, marked as answer]
Hi,

You should edit uptrace.yml file. See https://uptrace.dev/get/config.html#managing-projects.

**Reply by rupeshkumar1995** [reply]
Thanks its resolved now.
